utils/utils.cfg

```diff
@@ -1837,3 +1837,89 @@ $item_info.description"
         [/clear_menu_item]
     [/event]
 #enddef
+
+#define EVASION ABILITY AVOID_TERRAIN
+    [event]
+        name=attacker_misses
+        id=evasion_taunt
+        [filter_second]
+            ability={ABILITY}
+        [/filter_second]
+        [set_variables]
+            name=msgs
+            [value]
+                message=_"Hahaha, you think you can match my dexterity, $unit.name|?"
+            [/value]
+            [value]
+                message=_"Hahaha, nobody can match my dexterity!"
+            [/value]
+            [value]
+                message=_"Missed me by that <span font='oblique'>that</span> much"   # TV show Get Smart
```

Comment:
I would not bother with the comments where do these quotes come from, but if you want...

Reply:
I put those in there for translators, in case a phrase could be translated more than one way and they didn't catch the reference.

Reply:
Except for film references, they would not be able to use the references in the target languages. They'd have to find some other references instead.

Reply:
True, but at least I've attempted to convey what the quote refers to in en_US.  For example, there's a special taunt when Efraim misses L_evil, which I picked due to their prior dialog.  It's not important to translate verbatim, but to understand why she's saying what she's saying.

If it doesn't translate and/or they come up with a replacement that has no relation to the original reference, that's fine IMO.  And even if I didn't like it, I won't be able to read it anyway.

Reply:
All right.